### PR TITLE
Finished chapter computational_science.md

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/dietpi_docs_todo.md [hunk 1]
```diff
@@ -211,3 +213,26 @@ Check for this:
...
+
+- Let's Encrypt: <https://github.com/MichaIng/DietPi/wiki/Let's-Encrypt---Certbot>
+- TLS-SSL: <https://github.com/MichaIng/DietPi/wiki/TLS-SSL>
+- Security recommendations: <https://github.com/MichaIng/DietPi/wiki/Security-recommendation>
+- Fail2Ban: <https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=30#p452>
+
+## 12 Folding@Home
+
+Q: Who knows, whether the issue ***Known limitation of using only little computational power*** is already present?  
+See <https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=130#p13704> resp. [converted page](https://dietpi.com/docs/software/computational_science/).
```
Comment (MichaIng) [inline docs/dietpi_docs_todo.md:238]: Hmm, I used the Windows client recently and it uses all 4 CPU threads as long as the work unit can do that. Yes, generally work units may only utilise a single core/thread, or at least temporarily for certain steps, but that should be only true in very rare cases where for some reason no multi-threading is possible. I don't remember seeing this on any recent work unit.

Adding multiple folding slots should generally decrease overall performance, so I'd not recommend this anymore but trust that the work unit "developers" creation those in a way to utilise the system capabilities in an optimal way and as well assigning the work units that can be processed fine by the own system. E.g. my notebook has only builtin graphics, hence the GPU can only handle a very limited kind of tasks, hence I usually get CPU-only work units.

File: docs/software/computational_science.md [hunk 2]
```diff
@@ -2,11 +2,72 @@
...
+You can change the configuration with the following steps:
+
+1. Stop services with `dietpi-services stop`
+2. Edit */mnt/dietpi_userdata/fahclient/config.xml*
+3. Save changes and restart services with `dietpi-services start`
+
+### Logging information
+
+Folding@Home has its own log file located in */var/log/fahclient.log*.  
+You can display the log file with `cat /var/log/fahclient.log`.
```
Comment (MichaIng) [inline docs/software/computational_science.md:51]: The logging information is outdated. Seems I forgot to update the docs after moving to journal logs recently. Just did this now: https://dietpi.com/phpbb/viewtopic.php?p=13704#p13704

File: docs/dietpi_docs_todo.md (hunk 1 above)
Comment (StephanStS) [reply docs/dietpi_docs_todo.md:238]: @MichaIng: Then I will kick this section out of the docu.

File: docs/software/computational_science.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/computational_science.md:51]: Ok, I will use this.

**Pull request merged**